packages/support/resources/views/components/icon-button.blade.php

```diff
@@ -45,10 +45,12 @@
         'bg-warning-500/10' => $color === 'warning',
     ]);
 
-    $hasLoadingIndicator = filled($attributes->get('wire:target')) || filled($attributes->get('wire:click')) || (($type === 'submit') && filled($form));
+    $wireTarget = $attributes->whereStartsWith(['wire:target', 'wire:click'])->first();
+
+    $hasLoadingIndicator = filled($wireTarget) || ($type === 'submit' && filled($form));
 
     if ($hasLoadingIndicator) {
-        $loadingIndicatorTarget = html_entity_decode($attributes->get('wire:target', $attributes->get('wire:click', $form)), ENT_QUOTES);
+        $loadingIndicatorTarget = html_entity_decode($wireTarget ?: $form, ENT_QUOTES);
```

Comment:
Stricter to use `??` perhaps?

```suggestion
        $loadingIndicatorTarget = html_entity_decode($wireTarget ?? $form, ENT_QUOTES);
```

Reply:
I don't think so because the variable is always going to be set and covers empty string.

Reply:
Will it be set to an empty string or `null`?